package.json

```diff
@@ -2,6 +2,18 @@
   "name": "OpenSeadragon",
   "version": "2.1.0",
   "description": "Provides a smooth, zoomable user interface for HTML/Javascript.",
+  "keywords": ["image", "zoom", "pan"],
+  "homepage": "http://openseadragon.github.io/",
+  "bugs": {
+    "url": "https://github.com/openseadragon/openseadragon/issues"
+  },
+  "license": "BSD-3-Clause",
+  "files": ["build/openseadragon/"],
+  "main": "build/openseadragon/openseadragon.js",
```

Comment:
The standard is the non minified version. I think this is because in the majority of cases the `main` attribute is used by a build tool which will have its own minification step for production. This means you can then use the unminified version in development.

In fact the "new" standard is to include an `index.js` file which `require`s the modules main dist file and dependencies. I believe webpack and browserify promote this way of doing things.